portfolio.html

```diff
@@ -0,0 +1,231 @@
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item4">
+        <div class="item4a">
+          <img src="images/Snapshoot%20Portfolio4.png" alt="" />
+        </div>
+        <div class="item1b">
+          <h2>Uber Navigation</h2>
+          <div class="frame2">
+            <span>Uber</span>
+            <ul class="role">
+              <li>Lead Developer</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+           Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="grid-container2">
+      <div class="about-me">
+        <h1>
+          About Myself
+        </h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <a href="#" class="connect">LET’S CONNECT</a>
+        <div class="social-media2">
+          <ul>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/Linkedin%20icon.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/medium.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/angellist.svg" alt=""></a></li>
+          </ul>
+        </div>
+          <button type="button" class="btn"><a href="#">Get my resume</a></button>
+        </div>
+      </div>
+      <div class="skills">
+        <ul class="language">
+          <li>
+            <a class="languages" href="#">Languages<img class="arrow" src="images/Icon%20-%20Chevron%20-%20Down%20-%20Larger.svg" alt=""></a>
+            <ul class="sub-lang">
+              <li><a href=""><img src="images/Javascript.svg" class="animation" alt=""><span>JavaSript</span></a></li>
+              <li><a href="#"><img src="images/Html.svg" class="animation" alt=""><span>HTML</span></a></li>
+              <li><a href="#"><img src="images/CSS.svg" class="animation" alt=""/><span>CSS</span></a></li>
+            </ul>
+          </li>
+          <li><a class="frame" href="#">Frameworks <img class="arrow" src="images/frameworks_arrow.svg"></a></li>
+          <hr class="line2">
+          <li><a class="skl" href="#">Skills <img class="arrow2" src="images/frameworks_arrow.svg"></a></li>
+          <hr class="line2">
```

Comment:
- [ ] Please make sure the arrow position and font-family match the figma design as mentioned by the previous reviewer.
**Your project**
![r2](https://user-images.githubusercontent.com/74411135/229212145-3647018c-c656-46a8-8fa8-aa793c5f8552.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/229212274-5d30df4b-60ab-4996-b38f-8e35990e41f5.png)

This will make your design prettier which will improve the user experience 📱 💻 🖥️ 😃 😺